sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -66,7 +66,7 @@ import org.apache.spark.sql.internal.SQLConf
  * }}}
  */
 // scalastyle:on line.size.limit
-trait PlanStabilitySuite extends TPCDSBase with DisableAdaptiveExecutionSuite {
+trait PlanStabilitySuite extends TPCDSBase with TPCHBase with DisableAdaptiveExecutionSuite {
```

Comment:
It's a bit weird that `TPCDSBase` and `TPCHBase` are not at the same level (`TPCDSBase` is much more complicated). Can we do a bit more refactoring? Probably pull out some logic (e.g. `createTable`) from `TPCDSBase` to `PlanStabilitySuite` or a new `trait`.

Reply:
> It's a bit weird that `TPCDSBase` and `TPCHBase` are not at the same level (`TPCDSBase` is much more complicated). Can we do a bit more refactoring? Probably pull out some logic (e.g. `createTable`) from `TPCDSBase` to `PlanStabilitySuite` or a new `trait`.

How about current?